src/harness/sas_testcase.py

```diff
@@ -141,3 +141,24 @@ def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
     while not self._sas_admin.GetDailyActivitiesStatus()['completed']:
       time.sleep(10)
     signal.alarm(0)
+    
+  def TriggerFullActivityDumpAndWaitUntilComplete(self):
```

Comment:
I think you will need to take a SAS cert/key pair as input so that you can correctly call `self._sas.GetFullActivityDump()`. (While we could specify a default pair, I expect that ITS will want this to be configurable in every case so we may as well make it explicit.)

Reply:
ok I will add it, not added yet.

Reply:
I will add that as soos as possible.
for your comment: 
default values and max Eirp and eirpCapability  limits in the test the values of FCC max eirp injected in SAS UT are hardcoded in the test FAD.1 with 47 dB/10Mhz then I think we can have those limit values.

Reply:
It is done now,
I copied some method of FAD.2 and I noticed that the sasHarness section in the default config file  will contain something like:
"sasHarness": {
    "url": "https://localhost:9003/v1.2",
    "serverCert": "F:\\GitHubTest\\Spectrum-Access-System\\src\\harness\\certs\\server.cert",
    "sasTHName": "SAS-TH-2",
    "serverKey": "F:\\GitHubTest\\Spectrum-Access-System\\src\\harness\\certs\\server.key"
  }
then for  FAD.1 and FAD.2 we need to choose one option:
- use relative paths in getCertFilename method(I think, is the best one).
- change the default config file manually
- we can delete the default config file and ITS will generate it in their environement.